R/devmode.R

```diff
@@ -0,0 +1,354 @@
+#' Shiny Developer Mode
+#'
+#' @description \lifecycle{experimental}
+#'
+#'   Developer mode enables a number of options to make a developer's life
+#'   easier, like enabling non-minified JS and printing messages about
+#'   deprecated functions and options.
+#'
+#'   Please see the function descriptions for more details.
```

Comment:
What function descriptions?

Reply:
(... this whole file produces a single help file)

Reply:
Ah, ok